docs/Manual/06 Plans - Caching.md

```diff
@@ -78,15 +81,15 @@ AAA = plan.cache(AA, level=2)
 ```
 
 ## Multicaching
-Caches are defined with a key-slice `level`, and a higher-level key slice `trigger_level` can be specified as the trigger key-slice for copying multiple successive active blocks of elements to a local copy. These copied active blocks each have their layouts defined as usual, only the trigger level for copying them has been changed. Note that since active blocks are not mutually exclusive, this can result in the same element being copied into multiple locations in the local copy, however they will be in separate caches. Because of this, a `trigger_level` may only be specified on an `INPUT` or `CONST` array as Accera does not perform multicache write coherence.
+While caches are defined with a key-slice `level`, a higher-level key slice `trigger_level`  can be specified as the trigger key-slice for copying multiple successive active blocks of elements to a local copy. These copied active blocks have their layouts defined as usual, and only the trigger level for copying them has been changed. Since active blocks are not mutually exclusive, this can result in the same element being copied into multiple locations in the local copy, however, in separate caches. Therefore, a `trigger_level` may only be specified on an `INPUT` or `CONST` array as Accera does not support multicache write coherence.
```

Comment:
"this can result in the same element being copied into multiple locations as separate caches."